docker/docker-compose.1604.51.yaml

```diff
@@ -19,8 +19,8 @@ services:
       - MAX_ALLOCS_ALLOWED_1k_requests_interleaved=53000
       - MAX_ALLOCS_ALLOWED_1k_requests_noninterleaved=52000
       - MAX_ALLOCS_ALLOWED_client_server_h1_request_response=336000
-      - MAX_ALLOCS_ALLOWED_client_server_request_response=300000
-      - MAX_ALLOCS_ALLOWED_create_client_stream_channel=55000
+      - MAX_ALLOCS_ALLOWED_client_server_request_response=310000
```

Comment:
The regression here is the 'pending streams' dictionary in the multiplexer resizing (it isn't used for the frame-based stream channels).